core/ui/EditToolbar/delete.tid

```diff
@@ -3,11 +3,14 @@ tags: $:/tags/EditToolbar $:/tags/ViewToolbar
 caption: {{$:/core/images/delete-button}} {{$:/language/Buttons/Delete/Caption}}
 description: {{$:/language/Buttons/Delete/Hint}}
 
-<$button message="tm-delete-tiddler" tooltip={{$:/language/Buttons/Delete/Hint}} aria-label={{$:/language/Buttons/Delete/Caption}} class=<<tv-config-toolbar-class>>>
+\define focus-selector()
+[data-tiddler-title="$(cssEscapedTitle)$"].tc-tiddler-frame input
+\end
+<$button actions="""<$list filter="[list[$:/StoryList]after<currentTiddler>]"><$set name="cssEscapedTitle" value={{{ [<currentTiddler>escapecss[]] }}}><$action-sendmessage $message="tm-focus-selector" $param=<<focus-selector>>/></$set></$list><$action-sendmessage $message="tm-delete-tiddler"/>""" tooltip={{$:/language/Buttons/Delete/Hint}} aria-label={{$:/language/Buttons/Delete/Caption}} class=<<tv-config-toolbar-class>>>
```

Comment:
Can we make the actions attribute use a separate macro definition for the action text to make it easier to read?

Reply:
Thanks @Jermolene, it's more readable now